### ISSUE TITLE
e2edemo demo_dapp.ts check RollbackMessage infinity loop

### ISSUE THREAD

**Issue by R0bi7** [icon-project/btp2#36]
**Describe the issue**
Current implementation of demo_dapp.ts -> check RollbackMessage ends up in infinity loop between icon <> bsc rollback flow.

**To reproduce**
1. Send "revertMessage" from Icon to Bsc with revert and expect revert boolean flags set to true.
2. check RollbackMessage will end up in infinity loop if RollbackMessage is in same block as ResponseMessage

**Expected behavior**
RollbackMessage event should get picked up

**Solution**
Refactor implementation to always start checking for events from last receipt (on chain you are checking) blockheight and not from latest blockheight.

**Comment by sink772** [comment]
Yeah.... `ResponseMessage` and `RollbackMessage` are emitted by a same `handleRelayMessage` Tx execution, so finding the  `RollbackMessage` log should be from the same Tx receipt where the `ResponseMessage` was found.

I will modify the script like that.
